de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/filesystem/IntelliJProjectImplV2.java

```diff
@@ -71,6 +72,30 @@
     public IntelliJProjectImplV2(@NotNull final Module module) {
         this.module = module;
         this.moduleName = module.getName();
+        this.referencePoint = new IntelliJReferencePointImpl((IntelliJPathImpl)
+            this.getLocation());
+
+        ModuleRootManager moduleRootManager = ModuleRootManager
+            .getInstance(module);
+
+        final VirtualFile[] contentRoots = moduleRootManager.getContentRoots();
+
+        if (contentRoots.length == 0)
+            throw new IllegalArgumentException("module: " + module
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "module: " 4 times. ([squid:S1192](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=227&issues=AWYWtrKsvOWp7Yd6r4Gz&open=AWYWtrKsvOWp7Yd6r4Gz)

[ISSUE_KEY]: AWYWtrKsvOWp7Yd6r4Gz